### PR TITLE
Runtime 5.0.7 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

File: astro/runtime-release-notes.md [hunk 1]
```diff
@@ -5,10 +5,31 @@ id: runtime-release-notes
...
+## Astro Runtime 5.0.7
+
+- Release date: August 16, 2022
+- Airflow version: 2.3.3
+
+### Backported fixes from Apache Airflow
+
+Astro Runtime 5.0.5 includes several bug fixes and performance improvements that were backported from future Apache Airflow releases. Fixes include:
+
+- Fixed an issue where you could not import plugins as Python packages ([#25296](https://github.com/apache/airflow/pull/25296))
```
Comment (jedcunningham) [inline astro/runtime-release-notes.md:23]:
```suggestion
- Fixed an issue where webserver would crash loading plugins ([#25296](https://github.com/apache/airflow/pull/25296))
```
A little more clear what the actual impact was.

File: software/runtime-release-notes.md [hunk 2]
```diff
@@ -11,6 +11,25 @@ Astro Runtime is a Docker image built and published by Astronomer that extends t
...
 For instructions on how to upgrade, read [Upgrade Airflow on Astronomer Software](manage-airflow-versions.md). For general product release notes, go to [Software release notes](release-notes.md). If you have any questions or a bug to report, reach out to [Astronomer support](https://support.astronomer.io).
 
+## Astro Runtime 5.0.7
+
+- Release date: August 16, 2022
+- Airflow version: 2.3.3
+
+### Backported fixes from Apache Airflow
+
+Astro Runtime 5.0.5 includes several bug fixes and performance improvements that were backported from future Apache Airflow releases. Fixes include:
```
Comment (jedcunningham) [inline software/runtime-release-notes.md:21]: (All those same changes apply here too)

**Review by jedcunningham — APPROVED** [review]

File: astro/runtime-release-notes.md [hunk 3]
```diff
@@ -5,10 +5,31 @@ id: runtime-release-notes
...
 To upgrade Astro Runtime, see [Upgrade Astro Runtime](upgrade-runtime.md). For general product release notes, see [Astro Release Notes](release-notes.md). If you have any questions or a bug to report, contact [Astronomer support](https://support.astronomer.io).
 
+## Astro Runtime 5.0.7
+
+- Release date: August 16, 2022
+- Airflow version: 2.3.3
+
+### Backported fixes from Apache Airflow
+
+Astro Runtime 5.0.7 includes several bug fixes that were backported from future Apache Airflow releases. Fixes include:
```
Comment (paolaperaza) [inline astro/runtime-release-notes.md:21]:
```suggestion
Astro Runtime 5.0.7 includes the following bug fixes:
```

File: astro/runtime-release-notes.md [hunk 4]
```diff
@@ -5,10 +5,31 @@ id: runtime-release-notes
...
+
+- Fixed an issue where webserver would crash loading plugins ([#25296](https://github.com/apache/airflow/pull/25296))
+- Fixed the cleanup for zombie tasks from DAGs with parsing errors ([#25550](https://github.com/apache/airflow/pull/25550))
+- Fixed an issue where clearing a deferred task instance would not clear its `next_method` field ([#23929](https://github.com/apache/airflow/pull/23929))
+
```
Comment (paolaperaza) [inline astro/runtime-release-notes.md:26]:
```suggestion
- Fixed an issue where clearing a deferred task instance would not clear its `next_method` field ([#23929](https://github.com/apache/airflow/pull/23929))

For the benefit of Astro users, these change were backported from an Apache Airflow release that is not yet generally available.
```

File: astro/runtime-release-notes.md [hunk 5]
```diff
@@ -5,10 +5,31 @@ id: runtime-release-notes
...
+### Additional improvements
+
+- Upgraded `openlineage-airflow` to `0.12.0`, which includes support for Spark 3.3.0 and Apache Flink. For a list of all changes, see the [OpenLineage changelog](https://github.com/OpenLineage/OpenLineage/blob/main/CHANGELOG.md).
```
Comment (paolaperaza) [inline astro/runtime-release-notes.md:29]:
```suggestion
- Upgraded `openlineage-airflow` to version `0.12.0`, which includes support for Spark 3.3.0 and Apache Flink. For a list of all changes, see the [OpenLineage changelog](https://github.com/OpenLineage/OpenLineage/blob/main/CHANGELOG.md).
```

File: astro/runtime-release-notes.md [hunk 6]
```diff
@@ -5,10 +5,31 @@ id: runtime-release-notes
...
+- Upgraded `astronomer-providers` to `1.7.1`, which includes new deferrable operators and improvements to documentation. For more information, see the [Astronomer Providers changelog](https://github.com/astronomer/astronomer-providers/blob/1.7.1/CHANGELOG.rst).
```
Comment (paolaperaza) [inline astro/runtime-release-notes.md:30]:
```suggestion
- Upgraded `astronomer-providers` to version `1.7.1`, which includes new deferrable operators and improvements to documentation. For more information, see the [Astronomer Providers changelog](https://github.com/astronomer/astronomer-providers/blob/1.7.1/CHANGELOG.rst).
```

File: astro/runtime-release-notes.md [hunk 7]
```diff
@@ -5,10 +5,31 @@ id: runtime-release-notes
...
+- Upgraded `apache-airflow-providers-amazon` to `4.1.0`, which includes a bug fix for integrating with AWS Secrets Manager.
```
Comment (paolaperaza) [inline astro/runtime-release-notes.md:31]:
```suggestion
- Upgraded `apache-airflow-providers-amazon` to version `4.1.0`, which includes a bug fix for integrating with AWS Secrets Manager.
```

**Review by paolaperaza — APPROVED** [review]

File: astro/runtime-release-notes.md [hunk 8]
```diff
@@ -5,10 +5,33 @@ id: runtime-release-notes
...
+
+### Backported fixes from Apache Airflow
+
+Astro Runtime 5.0.7 includes the following bug fixes:
+
+- Fixed an issue where [plugins specified as a python package](https://airflow.apache.org/docs/apache-airflow/stable/plugins.html#plugins-as-python-packages) in an `entry_points` argument were incorrectly loaded twice by Airflow and resulted in an error in the Airflow UI. This bug was introduced in Airflow 2.3.3 ([#25296](https://github.com/apache/airflow/pull/25296))
+- Fixed an issue where zombie tasks were not properly cleaned up from DAGs with parse errors [#25550](https://github.com/apache/airflow/pull/25550))
+- Fixed an issue where clearing a deferred task instance would not clear its `next_method` field ([#23929](https://github.com/apache/airflow/pull/23929))
+
+These changes were backported from an Apache Airflow release that is not yet generally available. The bug fixes were also backported to Astro Runtime 5.0.5.
```
Comment (paolaperaza) [inline astro/runtime-release-notes.md:27]:
```suggestion
These changes were backported from Apache Airflow 2.3.4, which is not yet generally available. The bug fixes were also backported to Astro Runtime 5.0.5.
```

File: src/versions.js [hunk 9]
```diff
@@ -1,7 +1,7 @@
 export const siteVariables = {
   // version-specific
   cliVersion: '1.2.0',
```
Comment (paolaperaza) [inline src/versions.js:3]:
```suggestion
  cliVersion: '1.3.0',
```

**Pull request merged**